### ISSUE TITLE
Allow tips to be added via an API

### ISSUE THREAD

**Issue by Azeirah** [atom/background-tips#8]
I'd like to be able to add a tip or two for a plugin of mine.

Externalize the tips to a `.json` or `.cson`. Load tips from this storage file instead of using `module.exports`. Add a `.add(tip)` function to [background-tips.coffee](https://github.com/atom/background-tips/blob/master/lib/background-tips.coffee), which edits the storage file if the tip is unique (a plugin would add the same tip over and over again, which would clutter the background-tips plugin).

**Comment by bojidar-bg** [comment]
:+1:

**Comment by granteagon** [comment]
@Azeirah I agree.  I was actually going to do that.  Are you planning on doing it, or should I fork, implement, PR?
